src/Apps/Artist/Routes/AuctionResults/SingleAuctionResultPage/AuctionResultImage.tsx

```diff
@@ -0,0 +1,84 @@
+import { Flex, Image, NoImageIcon, ResponsiveBox } from "@artsy/palette"
+import { AuctionResultImage_auctionResult$key } from "__generated__/AuctionResultImage_auctionResult.graphql"
+import { graphql, useFragment } from "react-relay"
+
+export const MAX_DIMENSION = 400
+
+interface AuctionResultImageProps {
+  auctionResult: AuctionResultImage_auctionResult$key
+}
+
+export const AuctionResultImage: React.FC<AuctionResultImageProps> = ({
+  auctionResult,
+}) => {
+  const { images, title } = useFragment(
+    auctionResultImageFragment,
+    auctionResult
+  )
```

Comment:
🎉 i think this is the first use of hooks. We haven't used them much in force, yet.